lib/index.js

```diff
@@ -54,6 +54,8 @@ SaaSquatch.prototype.identify = function(identify) {
   var paymentProviderId = identify.proxy('traits.paymentProviderId');
   var accountStatus = identify.proxy('traits.accountStatus');
   var referralCode = identify.proxy('traits.referralCode');
+  var userReferralCode = identify.proxy('traits.userReferralCode');
```

Comment:
cc @sperand-io on this comment:

Doesn't this encourage users to pass a saasquatch-specific option in user traits? Should we have users pass this via `context.integrations`, instead?

(I realize this is following example, but just to understand)